tsconfig.build.json

```diff
@@ -4,6 +4,6 @@
     "allowJs": false,
     "outDir": "tmp",
     "declaration": true,
-    "declarationDir": "./"
+    "declarationDir": "./declarations"
```

Comment:
Is there a specific reason for this? I liked having the declarations next to the compiled files, so it's easier to navigate to the .js file when ctrl+clicking on a file in your IDE (that will end up at the declaration).

Reply:
Yeah, the TS compiler now seems to add the directory specified in declarationDir to the "exclude" option, resulting in no source files being picked up by the compiler. I can try to find a way around this. Do you know of any?

Reply:
Nevermind, fixed it by adding an explicit `"exclude": []`

Reply:
We could output the declarations in `./tmp/declarations`, and them move them to the root from there after compilation?